### DISCUSSION TITLE
or-tools can be compiled so library with android system?

### DISCUSSION THREAD

**Discussion by jjqcat** [google/or-tools#4126] [Feature requests]
**What language and solver does this apply to?**
 C++ / CP-SAT, Routing, Linear Solver

**Describe the problem you are trying to solve.**
I want to compile it into so library with android studio IDE, would it plan to support in future?

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Mizux** [comment]
did you try to use the cmake NDK cross toolchain to build ortools against the android ecosystem ?

**Comment by rma350** [comment]
Internally, we have successfully compiled at least linear solver + glop for android using bazel (internal bazel is not 100% the same).
